Tests/Unit/DSL/SearchTest.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ElasticsearchBundle\Tests\Unit\DSL;
+
+use ONGR\ElasticsearchBundle\DSL\Search;
+use ONGR\ElasticsearchBundle\DSL\Sort\AbstractSort;
+
+class SearchTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * Returns a mock object of AbstractSort class.
+     *
+     * @return \PHPUnit_Framework_MockObject_MockObject
+     */
+    private function getAbstractSortMock()
+    {
+        $mock = $this->getMockBuilder('ONGR\ElasticsearchBundle\DSL\Sort\AbstractSort')
+            ->setConstructorArgs(['field', AbstractSort::ORDER_ASC, AbstractSort::MODE_AVG])
+            ->getMock();
+
+        return $mock;
+    }
+
+    /**
+     * Returns a mock object of AbstractSuggester class.
+     *
+     * @return \PHPUnit_Framework_MockObject_MockObject
+     */
+    private function getAbstractSuggesterMock()
+    {
+        $mock = $this->getMockBuilder('ONGR\ElasticsearchBundle\DSL\Suggester\AbstractSuggester')
+            ->setConstructorArgs(['', ''])
+            ->getMock();
+
+        return $mock;
+    }
+
+    /**
+     * Tests addSort method.
+     */
+    public function testAddSort()
+    {
+        $search = new Search();
+        $this->assertEmpty($search->getSorts());
```

Comment:
When test has more than one assertion try to enter a third parameter (message) so that it would be easier to know later which assertion failed. **check other places also**